README.md

```diff
@@ -102,7 +104,7 @@ sudo apt-get install racket nodejs ant checkstyle testng
 To then run all the tests, run:
 
 ```bash
-test/bin/test-all
+make test-all
```

Comment:
The tests fail for me

```
$ make test-all
test/bin/test-all
Waxeye Parser Generator v0.8.1
generated: tmp/generated-parsers/c/json_parser.h
generated: tmp/generated-parsers/c/json_parser.c
Waxeye Parser Generator v0.8.1
generated: tmp/generated-parsers/c/templ_parser.h
generated: tmp/generated-parsers/c/templ_parser.c
Waxeye Parser Generator v0.8.1
generated: tmp/generated-parsers/c/waxeye_parser.h
generated: tmp/generated-parsers/c/waxeye_parser.c
test-grammars: PASS
Waxeye Parser Generator v0.8.1
generated: tmp/generated-parsers/c/env1_parser.h
generated: tmp/generated-parsers/c/env1_parser.c

TypeError: Object 1 has no method 'codePointAt'
    at WaxeyeParser.move (/path/to/waxeye/src/javascript/waxeye.js:388:42)
    at WaxeyeParser.match (/path/to/waxeye/src/javascript/waxeye.js:516:34)
    at WaxeyeParser.parse (/path/to/waxeye/src/javascript/waxeye.js:522:21)
    at TestEnv.match (/path/to/waxeye/test/javascript/lib.js:44:50)
    at TestEnv.getTestOutput (/path/to/waxeye/test/javascript/lib.js:12:17)
    at /path/to/waxeye/test/javascript/test.js:20:30
    at Array.map (native)
    at Object.<anonymous> (/path/to/waxeye/test/javascript/test.js:14:14)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
test-javascript: FAIL
test-c: PASS
Waxeye Parser Generator v0.8.1
generated: tmp/generated-parsers/java/jsonType.java
generated: tmp/generated-parsers/java/jsonParser.java
Waxeye Parser Generator v0.8.1
generated: tmp/generated-parsers/java/templType.java
generated: tmp/generated-parsers/java/templParser.java
Waxeye Parser Generator v0.8.1
generated: tmp/generated-parsers/java/waxeyeType.java
generated: tmp/generated-parsers/java/waxeyeParser.java
Could not load definitions from resource com/puppycrawl/tools/checkstyle/ant/checkstyle-ant-task.properties. It could not be found.

BUILD FAILED
/path/to/waxeye/build/ant.xml:143: Problem: failed to create task or type checkstyle
Cause: The name is undefined.
Action: Check the spelling.
Action: Check that any custom tasks/types have been declared.
Action: Check that any <presetdef>/<macrodef> declarations have taken place.


Total time: 3 seconds
test-java: FAIL
Waxeye Parser Generator v0.8.1
generated: tmp/generated-parsers/java/env1Type.java
generated: tmp/generated-parsers/java/env1Parser.java
Waxeye Parser Generator v0.8.1
generated: tmp/generated-parsers/javascript/json_parser.js
Waxeye Parser Generator v0.8.1
generated: tmp/generated-parsers/javascript/templ_parser.js
Waxeye Parser Generator v0.8.1
generated: tmp/generated-parsers/javascript/waxeye_parser.js
Waxeye Parser Generator v0.8.1
generated: tmp/generated-parsers/javascript/env1_parser.js
Waxeye Parser Generator v0.8.1
generated: tmp/generated-parsers/python/json_parser.py
Waxeye Parser Generator v0.8.1
generated: tmp/generated-parsers/python/templ_parser.py
Waxeye Parser Generator v0.8.1
generated: tmp/generated-parsers/python/waxeye_parser.py
Waxeye Parser Generator v0.8.1
generated: tmp/generated-parsers/python/env1_parser.py
Waxeye Parser Generator v0.8.1
generated: tmp/generated-parsers/racket/json-parser.rkt
Waxeye Parser Generator v0.8.1
generated: tmp/generated-parsers/racket/templ-parser.rkt
Waxeye Parser Generator v0.8.1
generated: tmp/generated-parsers/racket/waxeye-parser.rkt
Waxeye Parser Generator v0.8.1
generated: tmp/generated-parsers/racket/env1-parser.rkt
Waxeye Parser Generator v0.8.1
generated: tmp/generated-parsers/ruby/json_parser.rb
Waxeye Parser Generator v0.8.1
generated: tmp/generated-parsers/ruby/templ_parser.rb
Waxeye Parser Generator v0.8.1
generated: tmp/generated-parsers/ruby/waxeye_parser.rb
Waxeye Parser Generator v0.8.1
generated: tmp/generated-parsers/ruby/env1_parser.rb
test-generate-parsers: PASS
FAIL
make: *** [test-all] Error 123
```

Reply:
Your version of NodeJS is too old, codePointAt has been there since Node v4